how-to/workspace-platform-starter/public/common/views/frame/preload.js

```diff
@@ -0,0 +1,11 @@
+if (window == window.top) {
```

Comment:
Should this be ===, and why didn't lint pick this up. Looks like preload.js is excluded, have pushed a change to .eslint config so now your linting fails :)

Reply:
Updated to only run the logic if it isn't top and removed the return as it wasn't a function.